packages/autocomplete-core/src/utils/__tests__/isChrome.test.ts

```diff
@@ -0,0 +1,119 @@
+import { isChrome } from '../isChrome';
+
+describe('isChrome', () => {
+  describe('Android', () => {
+    describe('Mobile/tablet with Chrome', () => {
```

Comment:
Same for other suites.

```suggestion
    describe('Handheld devices with Chrome', () => {
```

Reply:
I actually better understand "Mobile/tablet".